dictionaries/top-10000-english-words.json

```diff
@@ -9020,7 +9020,7 @@
 "PHOEPLT/UPL": "momentum",
 "TPEUPB": "fin",
 "WAOEURG": "wiring",
-"KWR*EU/O*/SP-S": "io",
+"KWR*EU/O*": "io",
```

Comment:
Given `"KWR*EU": "{i^}{-|}",`, this would probably be better as `"*EU/O*": "io",`